src/google_map_markers.js

```diff
@@ -320,20 +327,40 @@ export default class GoogleMapMarkers extends Component {
           ...latLng,
         };
 
+        if (child.props.markerPosition) {
+          const markerPosition = child.props.markerPosition;
+          if (typeof markerPosition === 'string') {
+            const [left, top] = markerPosition.split(' ');
+            if (left && top) {
+              this.markerPosition = { transform: `translate(${left},${top})` };
+            } else {
+              console.warn(
+                `markerPosition expects a two values separated by a single whitespace`
+              );
+            }
+          } else {
+            console.error(
+              `markerPosition expects a value of type string, got ${typeof markerPosition} instead.`
+            );
+          }
+        }
+
         return (
           <div
             key={childKey}
             style={{ ...style, ...stylePtPos }}
             className={child.props.$markerHolderClassName}
           >
-            {React.cloneElement(child, {
-              $hover: childKey === this.state.hoverKey,
-              $getDimensions: this._getDimensions,
-              $dimensionKey: childKey,
-              $geoService: this.props.geoService,
-              $onMouseAllow: this._onMouseAllow,
-              $prerender: this.props.prerender,
-            })}
+            <div style={{ ...markerWrapperStyles, ...this.markerPosition }}>
```

Comment:
No need of extra div, it already exists above, you should merge the styles to `style` and not create a new one.